src/mono/wasm/debugger/debugger.md

```diff
@@ -0,0 +1,23 @@
+# Mono Wasm Debugger
+
+## Native debugging
+
+It's possible to debug native code and managed code using the `BrowserDebugProxy` project.
+
+Steps:
+- Install C/C++ DevTools Support (DWARF) extension on chrome.
+- Enable DWARF support: Open DevTools, click on the Engine, click on experiments and enable WebAssembly Debugging: Enable DWARF support.
```

Comment:
Probably most people would understand but I at first was not sure what Engine might mean. It's about Settings, right? 
![image](https://user-images.githubusercontent.com/32700855/170646447-b05d6b9b-18eb-4600-b0e4-5d5909fa8abb.png)